Code/Client/src/components/Game/game.js

```diff
@@ -69,13 +69,36 @@ class Game extends Component {
             return;
         
         setUpSocketHandlers(this);
-
     }
 
 
+
     onJoinTeam(teamNumber) {
         //notify server
-        this.state.socket.emit('join_team',this.state.userName,teamNumber);
+        this.state.socket.emit('join_team',teamNumber);
+    }
+
+    
+    submitNote(e){
+        e.preventDefault();
```

Comment:
What will happen here?

Reply:
you'll see

Reply:
I thought of allowing room participants to submit their own custom notes to join the notes pool.